docs/install-and-run.md

```diff
@@ -121,86 +132,88 @@ In cases where CocoaPods is not compatible with your project, you can add EarlGr
 
-    ```
-    Project → Test Target → Build Phases → Link Binary With Libraries → + (Add Sign) → EarlGrey.framework`
-    ```
+```
+Project → Test Target → Build Phases → Link Binary With Libraries → + (Add Sign) → EarlGrey.framework`
+```
 
-  3. Add EarlGrey as a Target Dependency to the Test Target:
+3. Add EarlGrey as a Target Dependency to the Test Target:
 
-    ```
-    Project → Test Target → Build Phases → Target Dependencies → + (Add Sign) → EarlGrey
-    ```
+```
+Project → Test Target → Build Phases → Target Dependencies → + (Add Sign) → EarlGrey
+```
 
-     The Test Target’s Build Phases should now look similar to this:
+The Test Target’s Build Phases should now look similar to this:
 
-     <img src="images/image06.png" width="450">
+<img src="images/image06.png" width="450">
 
-  4. Turn off Bitcode as it is not supported by EarlGrey (yet) by setting **Enable Bitcode** to **NO** in the Build Settings of the Test Target.
+4. Turn off Bitcode as it is not supported by EarlGrey (yet) by setting **Enable Bitcode** to **NO** in the Build Settings of the Test Target.
 
-  5. You must add environment variables in the Test Target's Scheme to inject the EarlGrey framework. To do so, go to **The Test Target → Edit Scheme → Test Action** and then deselect **Use the Run action's arguments and environment variables**. Add the following details in the `Environment Variables`:
+5. You must add environment variables in the Test Target's Scheme to inject the EarlGrey framework. To do so, go to **The Test Target → Edit Scheme → Test Action** and then deselect **Use the Run action's arguments and environment variables**. Add the following details in the `Environment Variables`:
 
-           Key: `DYLD_INSERT_LIBRARIES`
-           Value:`@executable_path/EarlGrey.framework/EarlGrey`
+Key: `DYLD_INSERT_LIBRARIES`
+Value:`@executable_path/EarlGrey.framework/EarlGrey`
 
-     Make sure the `Expand Variables Based On` value points to the app under test. The Scheme should now look like this:<a name="scheme-changes"></a>
+Make sure the `Expand Variables Based On` value points to the app under test. The Scheme should now look like this:<a name="scheme-changes"></a>
 
-    <img src="images/image07.png" width="500">
+<img src="images/image07.png" width="500">
 
 #### Step 3: Attempt to build the app under test
 
 In Xcode, attempt to build the app under test. It should build without any errors. After EarlGrey is built, see the [Final Test Configuration](#final-test-configuration) section for additional customizations that you may need to get your tests to run.
 
-### Final Test Configuration <a name="final-test-configuration"></a>
+#### Step 4: Final Test Configuration <a name="final-test-configuration"></a>
 
 The EarlGrey tests are hosted from the application being tested. Make sure the test target is setup
 correctly to launch the app under test:
 
 1. Under the **General** tab:
-   * **Host Application** is set to the app under test.
```

Comment:
Adding a) etc. doesn't seem to indent this correctly. Just changed this to * everywhere.